documentation/Extension_Point_Scheduled_Jobs.md

```diff
@@ -0,0 +1,27 @@
+# Scheduled Jobs
+
+To schedule the execution of some logic, annotate the class implementing the logic with `@ScheduledTask` 
+and implement the `com.github.dynamicextensionsalfresco.schedule.Task` interface. 
+
+```java
+import com.github.dynamicextensionsalfresco.schedule.ScheduledTask;
+import com.github.dynamicextensionsalfresco.schedule.Task;
+
+@Component
+@ScheduledTask(name = "example", cron = "0/15 * * * * ?")
+public class ScheduledExampleTask implements Task {
+
+    private static final Logger logger = LoggerFactory.getLogger(ScheduledExampleTask.class);
+
+    @Autowired
+    private NodeService nodeService;
+
+    @Override
+    public void execute() {
+        logger.info("Executing '{}'", ScheduledExampleTask.class.getCanonicalName());
+    }
+}
+```
+ 
+> The `@ScheduledQuartzJob` annotation has been deprecated since Dynamic Extensions 2.0 and replace by the 
+> vendor neutral `@ScheduledTask` annotation. It still works on 2.0, but is scheduled for removing in later versions.
```

Comment:
Maybe give us a little more info on which scheduling system is used by default (still quartz)? I need to know which docs to look at, the syntax for e.g. Quartz cron is subtly different from the vichy cron syntax (which is what you find on your Linux distros).

Reply:
Since it is a vendor neutral annotation now, we should document the syntax.
If there ever is a switch to an incompatible system, DE should convert between the documented syntax and the format the new system requires

Reply:
I'm just adding that we support Quartz. It is unlikely that Alfresco will ever switch to a different format.